yt_dlp/extractor/rtvs.py

```diff
@@ -3,45 +3,96 @@
 
 from .common import InfoExtractor
 
+from ..utils import (
+    int_or_none,
+    parse_duration,
+    traverse_obj,
+    unified_timestamp,
+)
+
 
 class RTVSIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?rtvs\.sk/(?:radio|televizia)/archiv/\d+/(?P<id>\d+)'
+    _VALID_URL = r'https?://(?:www\.)?rtvs\.sk/(?:radio|televizia)/archiv/?.*/(?P<id>\d+)\??.*'
```

Comment:
did u mean this?
```suggestion
    _VALID_URL = r'https?://(?:www\.)?rtvs\.sk/(?:radio|televizia)/archiv(?:/\d+)?/(?P<id>\d+)/?(?:[#?]|$)'
```

Reply:
Probably, I cant find any link supported that does not match that re.
I found other links that are from playlist and live video/radio events. But I will not code that now.